resident-default.properties

```diff
@@ -1151,6 +1144,3 @@ resident-transliteration-workaround-for-eng-spa=eng-hin,hin-spa
 
 # Reg-processer-credential-partner-policy-url
 mosip.resident.reg-processer-credential-partner-policy-url=${config.server.file.storage.uri}registration-processor-credential-partners.json
-
-## Configuration for language dropdown
```

Comment:
why you are removing this property

Reply:
This property was created to show and hide the language dropdown but now depending on the optional language length we are showing the language dropdown. If there is only one language dropdown, it will be hidden if there is more than one language dropdown will be displayed

Reply:
okay